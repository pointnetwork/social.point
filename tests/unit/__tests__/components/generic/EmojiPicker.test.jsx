import React from "react";
import renderer from 'react-test-renderer';

import EmojiPicker from '../../../../../src/components/generic/EmojiPicker';

describe("EmojiPicker", () => {

    it('should render', () => {
        const component = renderer.create(
            <EmojiPicker></EmojiPicker>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});