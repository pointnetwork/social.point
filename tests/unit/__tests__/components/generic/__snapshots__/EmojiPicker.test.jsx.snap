// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmojiPicker should render 1`] = `<div />`;
